### DISCUSSION TITLE
Can we have an adapter for nextjs?

### DISCUSSION THREAD

**Discussion by theprantadutta** [inertiajs/inertia#607] [Ideas]
I think it's safe to say if anyone is looking for a react framework that can do image optimization, static site generation, server-side generation, and code splitting, and much more, I think it should be NextJS. 
Currently, I am working with laravel as a backend, and NextJS as a frontend. It works great.
But can we have an inertia adapter for NextJS so that we can make our life a little easier?

**Comment by haikyuu** [comment]
Doesn't the React adapter work already?

**Reply by reinink** [reply]
Correct, we already have a React adapter.

And I don't think it makes sense to use Inertia in a Next app. They are different paradigms entirely.

**Comment by aleqske** [comment]
Maybe @prantaDutta raised this with regards to SSR rendering which I believe Inertia will have in the future. I started using Inertia a recently and I'm really liking it. Currently, what available options are there to build an SEO friendly site with Inertia?
